### DISCUSSION TITLE
Open-Shell replaced by standard Windows 10 shell after each restart since last Windows update

### DISCUSSION THREAD

**Discussion by JaaVb** [Open-Shell/Open-Shell-Menu#1770] [Q&A]
Each time I restart the PC the Open-Shell interface is gone. I reinstall it, with the option repair and it's back again with all custom settings. This started happening after install of the latest Windows updates about 10 days ago.

Any ideas how I can make the Open-Shell interface permanent again? 
Or does it need another update op Open-Shell?

**Comment by Jamesstevens-3rd** [comment]
For the start menu check this registry key

`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Run`

if its not there add a new **String Value** and name it **Open-Shell Start Menu** then modify it with the **Value Data** `"C:\Program Files\Open-Shell\StartMenu.exe" -autorun`

for classic explorer open the **Control Panel** then open **Internet Options** click the Programs tab and Manage add-ons  enable both **Classic Explorer Bar** and **ExplorerBHO Class**

**Reply by JaaVb** [reply]
Thanks, I solved it!

String values were OK. 

However, for some reason the option "Start automatically for this user" was not selected. How this happened, no idea, I have not changed the settings in a long time...

**Reply by Rufus543** [reply]
I think the latest Windows update has affected something, as Openshell will not let me customise the taskbar since, regardless of reinstalling. 

The string value mentioned is there, i'm not using classic explorer so I can't comment on that. 

edit - Opening ExplorerPatcher and letting it update fixed this issue. Openshell then worked properly after this.
